lua/nvim-navbuddy/display.lua

```diff
@@ -298,6 +298,7 @@ end
 function display:show_preview()
 	vim.api.nvim_win_set_buf(self.right.winid, self.for_buf)
 
+	vim.api.nvim_win_set_option(self.right.winid, 'winhighlight', 'Normal:NavbuddyNormal,FloatBorder:NavbuddyFloatBorder')
```

Comment:
Good catch! Thanks 👍🏽 

Just wondering, could this be a bug in neovim itself? 🤔  I have observed that using the `nvim_win_set_buf` messes up the window options for no apparent reason.

Reply:
Yeah I'm not sure, maybe it is just choosing the default highlight groups for a floating window?